### DISCUSSION TITLE
v3.0.93

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#9560] [General]
## What's Changed
* Add missing panel layout translation for Danish by @prip in https://github.com/filamentphp/filament/pull/9544
* Clean up testing examples in docs by @bzy107 in https://github.com/filamentphp/filament/pull/9540
* add luxon adapter to Chart.js by @manhofmann in https://github.com/filamentphp/filament/pull/9514
* return null for deprecated `allowsDuplicates` by @wychoong in https://github.com/filamentphp/filament/pull/9536
* Amending `assertColumnExists` to accept `Closure` to test configuration by @titantwentyone in https://github.com/filamentphp/filament/pull/9506
* Support custom content width in panel provider by @wychoong in https://github.com/filamentphp/filament/pull/9552
* Customize form fields validation message by @wychoong in https://github.com/filamentphp/filament/pull/9528
* Add more missing icon aliases by @zepfietje in https://github.com/filamentphp/filament/pull/9527

## New Contributors
* @manhofmann made their first contribution in https://github.com/filamentphp/filament/pull/9514

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.0.92...v3.0.93

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.0.93'>v3.0.93</a>.</em>

**Comment by lloricode** [comment]
customize validation message 🚀🚀🚀
